src/idvc/dvc_interface.py

```diff
@@ -4087,12 +4087,12 @@ def CreateRunDVCPanel(self):
         widgetno += 1
 
         rdvc_widgets['run_ndof_label'] = QLabel(groupBox)
-        rdvc_widgets['run_ndof_label'].setText("Number of Degrees of Freedom")
+        rdvc_widgets['run_ndof_label'].setText("Number of Optimisation Parameters")
         
-        dof_text = "Defines the degree-of-freedom set for the final stage of the search.\nThe actual search process introduces degrees-of-freedom in stages up to this value.\n\
-Translation only suffices for a quick, preliminary investigation.\nAdding rotation will significantly improve displacement accuracy in most cases.\nReserve strain degrees-of-freedom for cases when the highest precision is required.\n\
+        dof_text = "Defines the optimisation parameters in the final stage of the search.\nThe actual search process introduces more optimisation parameters, i.e. degrees of freedom,  in stages up to this value.\n\
```

Comment:
From Brian:
Delete the sentence "The actual search process introduces more optimisation parameters, i.e. degrees of freedom,  in stages up to this value.\n\"